tests/transaction_tests.cpp

```diff
@@ -102,20 +102,60 @@ TEST_CASE("Transaction guard") {
 
     SECTION("insert, call make a storage to call an exception and check that rollback was fired") {
         auto countBefore = storage.count<Object>();
-        try {
-            auto guard = storage.transaction_guard();
+        SECTION("transaction_guard") {
+            try {
+                auto guard = storage.transaction_guard();
 
-            storage.insert(Object{0, "John"});
+                storage.insert(Object{0, "John"});
 
-            storage.get<Object>(-1);
+                storage.get<Object>(-1);
```

Comment:
Also, the object ID to retrieve should be _**0**_, no?

Reply:
can be 0, but any value below 0 is also suitable for non-existing object

Reply:
but I confess this code was written by me during epoch when I was sure that `0` is a valid ID in SQLite